packages/nest/typedoc.json

```
@@ -0,0 +1,8 @@
+{
+    "$schema": "https://typedoc.org/schema.json",
+    "name": "@openfeature/react-sdk",
```

probably need to update this
```suggestion
    "name": "@openfeature/nestjs-sdk",
```